README.md

```diff
@@ -10,55 +10,64 @@ participate in the Gradle uptodate checks.
 You can find out more details about the actual jOOQ source code generation in the
 [jOOQ documentation](http://www.jooq.org/doc/latest/manual/code-generation).
 
-# Plugin
+The plugin is hosted on the [Gradle Plugin portal](https://plugins.gradle.org/plugin/nu.studer.jooq).
 
-## General
-The jOOQ plugin automatically applies the Java plugin. Thus, there is no need to explicitly apply the Java plugin in
-your build script when using the jOOQ plugin.
+# Applying the plugin
 
-Depending on the type of database that is accessed to derive the jOOQ Java sources, the corresponding driver must
-be put on the plugin classpath.
+You can apply the plugin using the plugins DSL
 
-The jOOQ plugin is hosted at [Bintray's JCenter](https://bintray.com/etienne/gradle-plugins/gradle-jooq-plugin).
+```groovy
+plugins {
+    id 'nu.studer.jooq' version '1.0.6'
+}
+```
 
-## Gradle 1.x and 2.x
-To use the jOOQ plugin with versions of Gradle 1.x and 2.x, apply the plugin in your `build.gradle` script:
+Or using the buildscript block
 
 ```groovy
 buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'nu.studer:gradle-jooq-plugin:1.0.5'
-        classpath 'postgresql:postgresql:9.1-901.jdbc4' // database-specific JDBC driver
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
     }
+  }
+  dependencies {
+    classpath "nu.studer:gradle-jooq-plugin:1.0.6"
+  }
 }
-apply plugin: 'nu.studer.jooq'
+
+apply plugin: "nu.studer.jooq"
 ```
 
-## Custom jOOQ Version
-You can use the jOOQ plugin with any current, previous, or future version of jOOQ. Simply enforce the required version of the jOOQ libraries in your `build.gradle` script:
+# Defining your database drivers
+
+Depending on which database you are connecting to, you'll need to put the corresponding driver on the generator's classpath.
+
 ```groovy
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'nu.studer:gradle-jooq-plugin:1.0.5'
-        classpath 'postgresql:postgresql:9.1-901.jdbc4' // database-specific JDBC driver
-    }
-    configurations.classpath {
-        resolutionStrategy {                            // enforce a specific jOOQ version
-            forcedModules = [
-                'org.jooq:jooq:3.4.1',
-                'org.jooq:jooq-meta:3.4.1',
-                'org.jooq:jooq-codegen:3.4.1'
-            ]     
-        }
-    }
+dependencies {
+    jooqGeneratorClasspath 'postgresql:postgresql:9.1-901.jdbc4'
+}
+```
+
+# Specifying the jOOQ version and edition
+
+This plugin supports existing and future jOOQ versions. It also supports the different editions like open source, pro and trial.
+
+```groovy
+jooq {
+  version = '3.8.2' //the default
+  edition = 'OSS'   //the default
```

Comment:
We should list the exact strings used for all the versions as a reference. 'PRO' is probably easy to guess but just to be thorough.